casadm/cas_main.c

```diff
@@ -358,6 +363,7 @@ static cli_option stats_options[] = {
 	{'d', "io-class-id", "Display per IO class statistics", 1, "ID", CLI_OPTION_OPTIONAL_ARG},
 	{'f', "filter", "Apply filters from the following set: {all, conf, usage, req, blk, err}", 1, "FILTER-SPEC"},
 	{'o', "output-format", "Output format: {table|csv}", 1, "FORMAT"},
+	{'p', "full-path", "Display full path to disks"},
```

Comment:
I would be more specific here: "display by-id path to disks instead of short form /dev/sdx"